### PR TITLE
fix: remove steps and add fields to others

### PR DESCRIPTION
**Pull request by jonnigs**
# ...

Attach a link to issue if relevant

## What

* Remove step "Er meira en ár liðið..." and add a warning if the date selected is more than a year ago.
* Remove step "Skráningarskylt ökutæki"
* Add field to accident description to describe symptoms
* Add fields to accident description to note the first time a doctor was seen
* Add a field for job title where information about the injured are filled out
* Add informational box if the injured was injured doing sports and he is an employee of that sports club
* Treat sports+employee as a work accident rather than a sports accident
* Add job title if self reporting
* Adjust overview screen for new data and remove old

## Why

The flow was redesigned and simplified

## Screenshots / Gifs
Screenshot 2024-05-16 at 12 38 36
Screenshot 2024-05-16 at 12 37 56
Screenshot 2024-05-16 at 12 36 45

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

## Summary by CodeRabbit

- **New Features**
  - Enhanced the accident notification form with new fields for job title, sport accidents, accident symptoms, and doctor visit details.
  - Added alert messages for sport accidents and conditions for accidents older than a year.
  - Improved accident location conditions based on sport accidents and employees.

- **Improvements**
  - Updated messages for accidents older than a year and sport-related incidents.
  - Added new labels for job titles and doctor visit details.

- **Bug Fixes**
  - Corrected the default message for general name in the application.

- **Utilities**
  - Introduced a function to determine if a sports accident involving an employee should be considered a work accident.

### CODE REVIEW

File: libs/application/templates/accident-notification/src/forms/AccidentNotificationForm/aboutTheAccidentSection.ts [hunk 1]
```diff
@@ -63,68 +64,13 @@ import {
...
+import { FormValue } from '@island.is/application/types'
+import { isSportAccidentAndEmployee } from '../../utils/isSportAccidentAndEmployee'
 
 export const aboutTheAccidentSection = buildSection({
   id: 'accidentType.section',
   title: accidentType.general.sectionTitle,
   children: [
-    buildSubSection({
-      id: 'hindrances',
-      title: hindrances.general.sectionTitle,
```
Comment (obmagnusson) [inline libs/application/templates/accident-notification/src/forms/AccidentNotificationForm/aboutTheAccidentSection.ts:73]: Remove unused import

File: libs/application/templates/accident-notification/src/forms/AccidentNotificationForm/aboutTheAccidentSection.ts [hunk 2]
```diff
@@ -242,6 +188,14 @@ export const aboutTheAccidentSection = buildSection({
                 },
               ],
             }),
+            buildAlertMessageField({
+              id: 'attachments.injuryCertificate.alert',
+              title: application.general.warningTitle,
```
Comment (obmagnusson) [inline libs/application/templates/accident-notification/src/forms/AccidentNotificationForm/aboutTheAccidentSection.ts:211]: Shouldn't this text be specific to this use case not a general title and message?

File: libs/application/templates/accident-notification/src/forms/AccidentNotificationForm/aboutTheAccidentSection.ts [hunk 3]
```diff
@@ -1403,3 +1402,6 @@ export const aboutTheAccidentSection = buildSection({
     }),
   ],
 })
+function isProfessionalAthleteAndEmployee(formValue: FormValue): boolean {
```
Comment (obmagnusson) [inline libs/application/templates/accident-notification/src/forms/AccidentNotificationForm/aboutTheAccidentSection.ts:1423]: Address lint warnings

**Review by obmagnusson — CHANGES_REQUESTED** [review]
Nice just a couple of comments

File: libs/application/templates/accident-notification/src/forms/AccidentNotificationForm/aboutTheAccidentSection.ts (hunk 2 above)
Comment (jonnigs) [reply libs/application/templates/accident-notification/src/forms/AccidentNotificationForm/aboutTheAccidentSection.ts:211]: I'll move from general to labels

**Pull request merged**